testing/src/main/java/org/oppia/android/testing/EditTextInputAction.kt

```diff
@@ -43,4 +49,30 @@ class EditTextInputAction @Inject constructor(
       }
     }
   }
+
+  fun hasErrorText(@StringRes expectedErrorTextId: Int): Matcher<View> {
```

Comment:
Create a new utility file for these functions. Don't add them in the `EditTextInputAction.kt`, as this file is specific to EditText helper functions only.

Reply:
@anandwana001 
Can i create a class name **EditInputAction.kt** inside org.oppia.android.testing
and then reference hasErrorText and hasNoErrorText inside AddProfileActivityTest.kt , AdminAuthActivityTest.kt , AdminPinActivityTest.kt , PinPasswordActivityTest.kt

Reply:
Can we have a more understandable name which gives a sense that we are using helper function for `TextInputLayout`.

Reply:
can we use HelperTextInputLayout or TextInputLayoutHelper

Reply:
WDYT about `TextInputAction`?

Reply:
@anandwana001 
yes ,do i have to create it inside org.oppia.android.testing or somewhere else

Reply:
Here - https://github.com/oppia/oppia-android/tree/develop/testing/src/main/java/org/oppia/android/testing

This is where we keep all the helper class files.

Reply:
@anandwana001 
I created TextInputLayout.kt file there are few imports which are not in use so should I remove them.

Reply:
yes please, else you will not be able to push code as it is kotlin file formatting error.

Reply:
Also, please resolve the conflicts.